### PR TITLE
Change layout to grid

### PR DESCRIPTION
**Pull request by martinremis**
## Summary
Replacing the display from flexbox to grid.
I've used a wrapping div to show the green line. I tried to use something else, but couldn't find something useful.

## Screenshot
grid

## Trello Card
https://trello.com/c/C5d2p9Fe/12-redise%C3%B1o-de-la-p%C3%A1gina

### CODE REVIEW

File: scss/tech.scss [hunk 1]
```diff
@@ -1,45 +1,39 @@
...
+  display: grid;
+  grid-auto-flow: row;
+  grid-template-areas:
+    "main . ."
+    "main . .";
+  grid-template-columns: 250px 250px 250px;
+  grid-template-rows: 1fr 1fr;
+  justify-content: center;
+  justify-items: center;
+  margin: 32px 0px 100px 0px;
```
Comment (FrankIglesias) [inline scss/tech.scss]: when you have 0px as value just put a `0`

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

File: index.html [hunk 2]
```diff
@@ -14,11 +14,13 @@ <h1 class="nav-title">Developer Tools</h1>
     </div>
     <h1 class="title">Front-End Tools</h1>
     <div class="technologies">
-      <div class="card">
-        <img src="assets/react_icon.svg" class="tech-logo" alt="react icon"/>
-        <h3 class="text-medium">React</h3>
-        <h4 class="text-small bold">Release: 2013</h4>
-        <p class="text-small">ReactJS is an open-source (with caveats) JavaScript library for building user interfaces.</p>
+      <div class="main-area">
+        <div class="card">
```
Comment (FrankIglesias) [inline index.html]: `<div class="card main-area">`

File: index.html [hunk 3]
```diff
@@ -47,11 +49,13 @@ <h4 class="text-small bold">Release: 2015</h4>
     </div>
     <h1 class="title">Back-End Tools</h1>
     <div class="technologies">
-      <div class="card">
-        <img src="assets/rails_icon.png" class="tech-logo" alt="rails icon"/>
-        <h3 class="text-medium">Ruby on Rails</h3>
-        <h4 class="text-small bold">Release: 2005</h4>
-        <p class="text-small">Ruby on Rails, or Rails, is a server-side web application framework written in Ruby. Rails is an MVC framework, providing default structures for a database, a web service, and web pages.</p>
+      <div class="main-area">
```
Comment (FrankIglesias) [inline index.html]: same

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by martinremis** [comment]
Image of the navbar shadow and the new titles margin:
grid2

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**